### PR TITLE
feat: show team on issues view

### PR DESCRIPTION
**Pull request by FezVrasta**
## Description

The "Team" field is displayed when a issue is open.

I'm not sure how to add it to the issue creation form though, for some reason it doesn't even show up in the unknown fields, help would be appreciated. 

implements #5190

## Screencast

CleanShot 2023-03-20 at 12 27 12@2x

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by FezVrasta** [comment]
Thanks, I think in order to fetch the teams list I need the `read:field.option:jira` scope. Do you think you could add it to the app? I tried to add it to the autorizationRequest call but it complains because the scope must match with the ones configured on the Jira app

**Comment by thomaslombart** [comment]
Unfortunately, I don't think the API  you're calling (`/field/teams-add-on__team-issue-field/option`) will be accessible as this is an API reserved to Connect apps. Our app is an OAuth 2.0 app, so you'll probably want to look into [this set of APIs](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issue-custom-field-options/#api-group-issue-custom-field-options). The first call ([Get custom field option](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issue-custom-field-options/#api-rest-api-3-customfieldoption-id-get)) seems to need the `jira:work` which is already enabled. 

Otherwise, you can also try to look into the following API: https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-jql/#api-rest-api-3-jql-autocompletedata-suggestions-get. We use that one to retrieve sprints.

**Comment by FezVrasta** [comment]
thanks! everything should work now

File: extensions/jira/src/api/teams.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  value: string;
+  displayName: string;
+};
+export type Team = {
+  teamId: string;
+  displayName: string;
+};
+
+export async function getTeams() {
+  const response = await request<{ results: TeamResponse[] }>("/jql/autocompletedata/suggestions", {
```
[the remark this answers is not in the document]
Comment (thomaslombart) [reply extensions/jira/src/api/teams.ts:13]: Makes sense. It'll always be possible to add it afterward if necessary.

**Review by thomaslombart — COMMENTED** [review]
Could you extract the team dropdown form in its dedicated component? That way, it'll be possible to move the call to retrieve the teams in it so that there's not an unnecessary API call performed for users who don't have/need teams.

**Comment by FezVrasta** [comment]
> Could you extract the team dropdown form in its dedicated component? That way, it'll be possible to move the call to retrieve the teams in it so that there's not an unnecessary API call performed for users who don't have/need teams.

Done

**Review by thomaslombart — APPROVED** [review]
Awesome, thanks for the contribution! 🚀

**Pull request merged**